src/NCronJob/Execution/JobExecutor.cs

```diff
@@ -70,7 +70,7 @@ public async Task RunJob(JobRun run, CancellationToken stoppingToken)
         }
         catch (Exception exc) when (exc is not OperationCanceledException or AggregateException)
         {
-            LogExceptionHandlerError(runContext.JobType);
+            LogExceptionHandlerError(runContext.JobRun.JobDefinition.Type);
```

Comment:
How does `runContext.JobRun.JobDefinition.Type` differ now to `runContext.JobType` - if so (as this is customer-facing), wouldn't it be better to keep the old one?

Reply:
![image](https://github.com/user-attachments/assets/6aee86e0-dff0-4f52-beae-24259a72e085)

JobType is internal only.

Besides, this is a temporary sidestep removal until I get better insights about the logging.
Depending on your output on the naming related comment, we might be able to completely get rid of JobDefinition.Type (as an exposed property).